CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## [Unreleased]
 
+### Added
+
+- [`view_*` asynchronous builders have been added which provides being able to query from a specific block hash or block height](https://github.com/near/workspaces-rs/pull/218)
+
+### Changed
+
+- [Apart of the changes from adding `view_*` async builders, we have a couple breaking changes to the `view_*` functions](https://github.com/near/workspaces-rs/pull/218):
+  - `{Account, Contract, Worker}::view_state` moved `prefix` parameter into builder. i.e.
+    ```
+    worker.view_state("account_id", Some(prefix)).await?;
+    // is now
+    worker.view_state("account_id")
+        .prefix(prefix)
+        .await?;
+    // if prefix was `None`, then simply delete the None argument.
+    ```
+  - `view` function changed to be a builder, and no longer take in `args` as a parameter. It instead has been moved to the builder side.
+  - Changed `Worker::view_latest_block` to `Worker::view_block` as the default behavior is equivalent.
+  - `operations::Function` type no longer takes a lifetime parameter.
```

Comment:
Maybe worth also noting `CallTransaction` has one less lifetime for the same reason